src/foundry/client/data/documents/journal-entry-page.d.mts

```diff
@@ -98,6 +99,115 @@ declare global {
       }>,
     ): JournalEntryPage.JournalEntryPageHeading;
 
-    // TODO: More method overrides
+    protected override _createDocumentLink(
+      eventData: unknown,
+      options?: InexactPartial<{
+        /**
+         * A document to generate a link relative to.
+         */
+        relativeTo: ClientDocument;
+        /**
+         * A custom label to use instead of the document's name.
+         */
+        label: string;
+      }>,
+    ): string;
+
+    override _onClickDocumentLink(event: MouseEvent): this;
+
+    protected override _onUpdate(
+      changed: BaseJournalEntryPage.UpdateData,
+      options: DocumentModificationOptions,
+      userId: string,
+    ): void;
+
+    protected override _buildEmbedHTML(
+      config: TextEditor.DocumentHTMLEmbedConfig,
+    ): Promise<HTMLCollection | HTMLElement | null>;
+
+    protected override _createFigureEmbed(
+      content: HTMLElement | HTMLCollection,
+      config: TextEditor.DocumentHTMLEmbedConfig,
+      options?: TextEditor.EnrichmentOptions,
+    ): Promise<HTMLElement | null>;
+
+    /**
+     * Embed text page content.
+     * @param config        - Configuration for embedding behavior. This can include
+     *                        enrichment options to override those passed as part of
+     *                        the root enrichment process.
+     * @param options       - The original enrichment options to propagate to the embedded text page's
+     *                        enrichment.
+     * @returns
+     *
+     * @example Embed the content of the Journal Entry Page as a figure.
+     * ```@Embed[.yDbDF1ThSfeinh3Y classes="small right"]{Special caption}```
```

Comment:
There's rightfully a lint:
> tsdoc-code-fence-specifier-syntax: Error parsing code fence: The language specifier cannot contain backtick characters    

To address it please add newlines after the first \`\`\` and before the last \`\`\`